svrestarter.go

```diff
@@ -45,7 +45,10 @@ func (s *SvRestarter) Restart() error {
 		err                  error
 		restartDone          = make(chan struct{})
 		preemptionAcceptable = make(chan struct{})
+		start 				 = time.Now()
```

Comment:
With Go, it's helpful to set up your editor to run `gofmt` on save to automatically format code. You can run it manually too though like: `gofmt -s -w svrestarter.go`